webcomponents-lite.js

```diff
@@ -53,9 +53,9 @@
     'URL/URL.js',
     'WeakMap/WeakMap.js',
     'MutationObserver/MutationObserver.js',
+    'Template/Template.js',
```

Comment:
I chatted with Dan, and discovered that `build.json` pulls in ShadowDOM Polyfill, which has a separate `HTMLTemplate.js` implementation... so, I think `build-lite.json` is the only thing that needs to be modified?